FirebaseSegmentation/Sources/SEGContentManager.m

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "SEGContentManager.h"
+
+#import <FirebaseCore/FIRAppInternal.h>
+#import <FirebaseInstanceID/FIRInstanceID.h>
+#import "FIRSegmentation.h"
```

Comment:
I think we should be using fully qualified imports. @paulb777 to confirm

Reply:
Yes - we should now use repo-relative imports. See the `s.pod_target_xcconfig` section at https://github.com/firebase/firebase-ios-sdk/blob/master/AddNewPod.md